### ISSUE TITLE
Bug with increase/decrease of speed.

### ISSUE THREAD

**Issue by Gryogor** [ros-teleop/teleop_twist_keyboard#26]
Increase and decrease in speed commands will also repeat the last issued movement command (move, strafe, rotate, stand still).

**Comment by trainman419** [comment]
Yes; when you increase speed a new movement command is sent, so that the robot moves at the new speed.

Why do you think this is a bug? What behavior were you expecting?

**Comment by Gryogor** [comment]
To just increase the speed without any movement. It is not stated that it will increase the speed AND repeat the last movement, so other behavior is confusing.

**Comment by wernersbacher** [comment]
In my understanding, the twist message is always carrying the whole movement information (state) in the any moment. So it shouldn't really matter if the message gets  sent once, ten or hundred times to your robot. So as you change the speed on your keyboard, it will just resent the new state.
